test/cases/compile_errors/enum_field_value_references_enum.zig

```diff
@@ -12,4 +12,5 @@ const D = 1;
 // backend=stage2
 // target=native
 //
-// :1:5: error: dependency loop detected
+// :2:13: error: enum 'tmp.Foo' has no member named 'B'
```

Comment:
Uhh, this isn't really the point of this test, we're now erroring out too soon to actually test the dependency loop. Maybe add a field `B` to `Foo` to get back to the old error.